src/main/java/ie/ait/bteam/drcare/data/model/User.java

```diff
@@ -36,13 +36,14 @@
 	@Getter
 	@Setter
 	@Column(unique = true)
+	@NotNull
```

Comment:
Patient doesn't need to have email, so remove this pls.

Reply:
okay done...